### PR TITLE
Windows SecureCRT Lab/Session Generation

### PR DESCRIPTION
**Pull request by kellychristians**
New SecureCRT windows lab/session generation script.

### CODE REVIEW

File: scripts/breakout-to-secureCRT-windows/readme.rnd [hunk 1]
```diff
@@ -0,0 +1,30 @@
+This tool will generate a lab.yaml file with the breakout tool.
+
+I wanted something that was simple and didn't need alot of modules loaded.
+I decided to make use of the yaml file the breakout tool will generate and use that as a basis for generating my secureCRT session files.
+
+
+It will discover your secureCRT configuration folders from the windows registry.
+IT then purges out anything inside the sessions\_CML folder (if it exists)
```
Comment (xorrkaz) [inline scripts/breakout-to-secureCRT-windows/readme.rnd:8]: Typo

File: scripts/breakout-to-secureCRT-windows/readme.rnd
Comment (xorrkaz) [inline scripts/breakout-to-secureCRT-windows/readme.rnd:1]: We typically use markdown for READMEs.  Can you convert this?

File: scripts/breakout-to-secureCRT-windows/__init__.py
Comment (xorrkaz) [inline scripts/breakout-to-secureCRT-windows/__init__.py:1]: This file isn't needed since you aren't producing a module.

**Review by xorrkaz — CHANGES_REQUESTED** [review]
Thanks for your contribution.  I have a few comments.

File: scripts/breakout-to-secureCRT-windows/readme.rnd (hunk 1 above)
Comment (kellychristians) [reply scripts/breakout-to-secureCRT-windows/readme.rnd]: Thanks, i've rewritten the readme

File: scripts/breakout-to-secureCRT-windows/readme.rnd
Comment (kellychristians) [reply scripts/breakout-to-secureCRT-windows/readme.rnd:1]: Thanks, i've rewritten the readme

File: scripts/breakout-to-secureCRT-windows/__init__.py
Comment (kellychristians) [reply scripts/breakout-to-secureCRT-windows/__init__.py:1]: i've removed

**Review by xorrkaz — APPROVED** [review]
LGTM

**Pull request merged**